packages/interpretations/src/components/html-editor/CKEditor.js

```diff
@@ -0,0 +1,309 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { noop, some } from 'lodash/fp';
+import log from 'loglevel';
+
+/* Additions from standard CKEditor:
+
+    - Tweak internal editor CSS (CKEDITOR.addCss)
+    - Tweak external (toolbar, dialogs) CSS
+    - Add cke_smiley class to dialog to allow custom styling.
+    - Dock position of smiley dialog to button
+    - Hide dialogs on click outside dialog.
+*/
+
+class WrappedEditor {
+    constructor(editor) {
+        this.editor = editor;
+    }
+
+    getValue() {
+        return this.editor.getData();
+    }
+
+    setCursorAtEnd() {
+        setTimeout(() => {
+            this.editor.focus();
+            const range = this.editor.createRange();
+            const selection = this.editor.getSelection();
+            if (range && selection) {
+                range.moveToElementEditEnd(range.root);
+                this.editor.getSelection().selectRanges([range]);
+            }
+        }, 300);
+    }
+
+    getPosition(offset = null) {
+        const range = this.editor.getSelection().getRanges()[0];
+        const bbox = range ? range.startContainer.$.parentNode.getBoundingClientRect() : {top: 0, left: 0};
+        const iframe = this.editor.container.$.getElementsByTagName("iframe")[0];
+        const iframeBbox = iframe.getBoundingClientRect();
+        const pos = { top: bbox.top + iframeBbox.top, left: bbox.left + iframeBbox.left };
+        return offset ? { top: pos.top + offset.top, left: pos.left + offset.left } : pos;
+    }
+
+    getCurrentWord() {
+        const range = this.editor.getSelection().getRanges()[0];
+        if (range) {
+            const text = range.startContainer.type === CKEDITOR.NODE_TEXT ? range.startContainer.getText() : "";
```

Comment:
How is the global injected into the app? Does the app have to do something like: https://github.com/dhis2/maintenance-app/blob/bd75c6855eee603ce24fbd7b31464b6e1071d3c2/webpack.config.js#L124 to make it work?

If so, it needs to be added to the README.

Reply:
Exactly, we took maintenance-app as reference: https://github.com/dhis2/maps-app/pull/13/files

Added to README: c466eea